draft-ietf-tls-dtls-rrc.md

```diff
@@ -87,6 +92,10 @@ DTLS 1.2 {{!I-D.ietf-tls-dtls-connection-id}} and for DTLS 1.3
 {{!I-D.ietf-tls-dtls13}}.  The presentation language used in this document is
 described in Section 4 of {{!RFC8446}}.
 
+This document reuses the definition of "anti-amplification limit" from
+{{?RFC9000}} to mean three times the amount of data received from an
+unvalidated address.
```

Comment:
My feeling, depending on the upper layer protocol and use-case, for very small messages received, I would consider to have also/additionally an absolute threshold. In term of the attack, it should also be clear, if the size to consider is the "dtls-payload", the "udp-payload", or the "ip-payload". Again, for assumed very small IoT messages, that may make the difference.

Reply:
> My feeling, depending on the upper layer protocol and use-case, for very small messages received, I would consider to have also/additionally an absolute threshold.

I think there is already an (implicit) upper limit, which is 3xPMTU.

> In the scope of the mitigated DoS attack, it should also be clear, if the size to consider is the "dtls-payload", the "udp-payload", or the "ip-payload". Again, for assumed very small IoT messages, that may make the difference.

I don't think we should make assumptions on the underlay.  This refers to data in DTLS Application Data records.  Definitely worth clarifying.